specification/dimensions/commitmentid.md

```diff
@@ -0,0 +1,32 @@
+# Commitment ID
```

Comment:
Can we change this to "Commitment Discount ID"? (Same for the name column.)

Reply:
You mean any references to "Commitment" should be changed to "Commitment Discount", right? (include filenames as well: `commitmentdiscountid.md`, etc.)

Reply:
Renamed.